ide/static/css/style.css

```diff
@@ -95,27 +113,37 @@ body {
 
 .topbar-col {
   flex: 1;
+  margin: 2px;
 }
 
 input#netName {
+  transition: 0.3s;
   border: none;
   position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
+  top: 15px;
+  right: 305px;
```

Comment:
Fix alignment of `netName` control it isn't placed in center of the canvas.

Reply:
Fixed 5e83dc8b707e27b72fe14415eb1d5097446bda6f.